connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -205,6 +205,9 @@ private void onCommitCompleted(Throwable error, long seqno) {
             } else {
                 log.debug("{} Finished offset commit successfully in {} ms",
                         this, time.milliseconds() - commitStarted);
+                if (committedOffsets != null) {
+                    lastCommittedOffsets = committedOffsets;
+                }
```

Comment:
Note that this does nothing if the supplied `committedOffsets` is null. We use this in most of the invocations of `onCommitCompleted` that are used in the various situations where the commit process fails.